### PR TITLE
Created a GET request to get mission

### PR DESCRIPTION
**Pull request by Celnet-hub**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -10,10 +10,13 @@ const xss = require('xss-clean');
...
 const globalErrorHandler = require('./controllers/errorController');
 
+// Require Routes
 const goalRouter = require('./routes/goalRoutes');
 const pluginInfoRouter = require('./routes/infoRoute');
+const missionRouter = require('./routes/missionRoute.js');
 const pingRouter = require('./routes/pingRoute');
-const sidebarRouter = require('./routes/sidebarRoute');
+// const sidebarRouter = require('./routes/sidebarRoute');
+const sidebarRouter = require('./routes/sidebarRoute.js');
```
Comment (Asylcreek) [inline app.js:19]: Please remove one of these

File: app.js [hunk 2]
```diff
@@ -50,6 +53,7 @@ app.use('/api/v1/goals', rateLimiter(), goalRouter);
 app.use('/ping', rateLimiter(), pingRouter);
 app.use('/sidebar', rateLimiter(), sidebarRouter);
 app.use('/info', rateLimiter(), pluginInfoRouter);
+app.use('/get_mission', missionRouter);
```
Comment (Asylcreek) [inline app.js:56]: Please change 'get_mission' to 'get-mission'

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (Celnet-hub) [reply app.js:19]: That's why it was comments. I don't know what did that

File: app.js (hunk 2 above)
Comment (Celnet-hub) [reply app.js]: Done

**Comment by Celnet-hub** [comment]
**Created a route to get the company's mission**

Click [link](https://goals.zuri.chat/api/mission/6135d087098aca7c1a5bc6a3) to test.

**Pull request merged**